### DISCUSSION TITLE
If automa can create random event to make it more like a human?

### DISCUSSION THREAD

**Discussion by Neal86** [AutomaApp/automa#1104] [Q&A]
Hi, everyone, can automa create a random event to make the workflow more like a human?
For example, 
1. I want a click action is delayed 2-9 seconds, so the action can be delayed 2/3/4/5/6/7/8/9 seconds in a workflow.
2. ENter different text(good post/nice video/funny) to a comment form, 

Thank you!

**Comment by Kholid060** [comment]
1. You can use the Delay block and input `{{ $randint(2000, 9000) }}` as the value. It will randomize the workflow delay between 2 to 9 seconds.
2. For this, you can input the list of text in the workflow global data like 
```json
{
  "texts": ["good post", "nice video", "funny"]
}
``` 
And input `{{ $randData([globalData.texts]) }}` as the value in the Forms block.

**Comment by Neal86** [comment]
Hi, thank you for your reply,
1. It works great, thank you!
2. I follow your instruction to add global data&form and the do test the
form block but it didn't work, I attached the workflow, can you please help
check?

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
23/01/16
21:27:10

Ahmad Kholid ***@***.***> 于2023年1月16日周一 18:57写道：

**Comment by Kholid060** [comment]
Can you resend the workflow file? Because I didn't receive any file

**Comment by Neal86** [comment]
Did you receive it?

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
23/01/17
下午01:18:48

Ahmad Kholid ***@***.***> 于2023年1月17日周二 00:00写道：

**Reply by Neal86** [reply]
Hi, Github doesn't support the file type, so I send it to your email, kindly check please.
![image](https://user-images.githubusercontent.com/52608308/213019821-e6efed11-84be-4b9b-a79b-956295eb0fc3.png)

**Comment by Neal86** [comment]
[TK 养号.automa.zip](https://github.com/AutomaApp/automa/files/10442518/TK.automa.zip)
I send the file by zip, kindly check please,
